deploy/tls/kustomization.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: metal3
+resources:
+- ../default
+
+configMapGenerator:
+- behavior: merge
+  literals:
+    - IRONIC_CACERT_FILE=/tls/ironic_ca.crt
+  name: ironic-bmo-configmap
+
+secretGenerator:
```

Comment:
Not sure what this means, but a certificate is not a secret.

Reply:
That's kustomize specific, to create a K8S secret with the certificate and key.